deploy/charts/venafi-kubernetes-agent/templates/configmap.yaml

```diff
@@ -0,0 +1,194 @@
+{{ if not .Values.config.configmap.name }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: agent-config
+  labels:
+    {{- include "venafi-kubernetes-agent.labels" . | nindent 4 }}
+data:
+  config.yaml: |-
+    server: {{ .Values.config.server | quote }}
+    venafi-cloud:
+      uploader_id: "no"
+      upload_path: "/v1/tlspk/upload/clusterdata"
```

Comment:
Linking back to https://github.com/jetstack/jetstack-secure/pull/458/files#r1691872028: why do we need to set an upload path explicitely when the default one should work out of the box?